client/app/scripts/superdesk-users/views/user-preferences.html

```diff
@@ -30,6 +33,34 @@
                     <span>{{ datelinePreview | previewDateline:datelineSource}} </span>
                 </div>
             </li>
+
+            <li class="fieldset">
+              <div class="legend">
+                {{ :: preferences['categories:preferred'].category }}
+              </div>
+
+              <div sd-info-item>
+                  <label translate>
+                    {{ :: preferences['categories:preferred'].label }}
+                  </label>
+                  <label translate>
+                    By checking the categories you are interested in, the system
+                    will only display them in a menu for setting the content
+                    item's categories (along with any of its existing categories).
+                  </label>
+
+                  <div ng-repeat="cat in categories track by cat.qcode">

```

Comment:
`sd-typeahead` directive would have been better instead of displaying all categories at once with check boxes.